test/kubernetes/e2e/features/deployer/suite.go

```diff
@@ -383,8 +388,8 @@ func awsStsClusterAssertion(testInstallation *e2e.TestInstallation) func(ctx con
 			g.Expect(socketAddr.GetAddress()).To(gomega.Equal(expectedStsUri))
 		}).
 			WithContext(ctx).
-			WithTimeout(time.Second * 10).
-			WithPolling(time.Millisecond * 200).
+			WithTimeout(time.Second * 30).
```

Comment:
I'm good with this change. I'm always a little nervous when we have timeouts that take 30 seconds (or some time frame beyond what I would expect for a change to propagate in a small environment) since those can be a signal of some other issue going on that we're masking.